cmds/core/printf/utils.go

```diff
@@ -0,0 +1,211 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+func interpret(
```

Comment:
in general, in Go, this kind of arg formatting is not really idiomatic. Please fit this declaration on one line.